src/applications/vaos/utils/selectors.js

```diff
@@ -37,6 +37,16 @@ export function getFormPageInfo(state, pageKey) {
   };
 }
 
+export const selectFacilities = state =>
+  selectPatientFacilities(state)?.filter(
+    f => !f.facilityId.startsWith('742'),
+  ) || null;
+
+export const selectCernerFacilities = state =>
+  selectFacilities(state)
```

Comment:
You could use `selectPatientFacilities` here, the logic around removing 742 is in `selectPatientFacilities` now.

Reply:
Done